packages/rum-core/src/domain/rumEventsCollection/action/trackClickActions.ts

```diff
@@ -80,17 +81,18 @@ export function trackClickActions(
 
   const { stop: stopActionEventsListener } = listenActionEvents<ClickActionBase>({
     onPointerDown: (pointerDownEvent) => processPointerDown(configuration, history, pointerDownEvent),
-    onClick: (clickActionBase, clickEvent, getUserActivity) =>
-      processClick(
+    onActionStart: (clickActionBase, startEvent, getUserActivity, getClickEventTimeStamp) =>
+      startClickAction(
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏`onActionStart` naming seems a bit confusing to me since:

- we are listening action events like pointer down above and here it seems more domain related
- we are actually starting the click action just below

What about `onStartEvent` instead?